Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04
```

Comment:
The docker image fails to run on an M1 system with an error as: `qemu-x86_64: Could not open '/lib64/ld-linux-x86-64.so.2': No such file or directory`

The fix to make it run was to change it to `FROM --platform=linux/amd64 ubuntu:22.04`:
```
diff --git a/Dockerfile b/Dockerfile
index 981fc4dcd..d267cf59d 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM --platform=linux/amd64 ubuntu:22.04
 LABEL maintainer="Chef Software, Inc. <docker@chef.io>"
 
 ARG VERSION=5.22.3
```